backend/routes/group.js

```diff
@@ -257,11 +257,13 @@ const getDbColOptions = function (req) {
 function listGroups(req, res, next) {
 
 	const dbCol = getDbColOptions(req);
-	const place = utils.APIInfo(req);
+	const place = utils.APIInfo(req);	
 
 	let groupList;
 	if (req.params.rid) {
 		groupList = Group.listGroups(dbCol, req.query, null, req.params.rid);
+	} else if (req.query.ids) {
```

Comment:
This needs to use the same `listGroups` function. As it stands you're ignoring the revision information if there's ID given,

List Groups do more than just getting the group information back and for consistency it should do it the same way.

You should add the id list to the list group method like how risks/issues are doing it

Reply:
I have moved the id lookup functionality into listGroups, to keep it the same as risks and groups.